src/simulate.jl

```diff
@@ -19,20 +23,30 @@ function simulate(series::Vector{T}, gas::Model{D, T}, H::Int, S::Int;
                     initial_params::Matrix{T} = stationary_initial_params(gas)) where {D, T}
     # Filter params estimated on the time series
     params = score_driven_recursion(gas, series; initial_params = initial_params)
-    biggest_lag = number_of_lags(gas)
-    params_simulation = params[(end - biggest_lag):(end - 1), :]
+    biggest_lag = ScoreDrivenModels.number_of_lags(gas)
```

Comment:
These prefixes are not necessary, I think you might have added them to test the code?

Reply:
yes, I am goingo to remove them